supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -130,6 +130,33 @@
 
 *See also*:
 
+[discrete]
+[[tls]]
+==== image:images/yes.png[yes] TLS (noun)
+*Description*: TLS is an initialism for Transport Layer Security. TLS is the successor to the Secure Sockets Layer (SSL) protocol. TLS is a cryptographic protocol that uses the Public Key Infrastructure (PKI) method to encrypt network traffic between two systems. PKI uses symmetric encryption during a TLS handshake process to authenticate the connection between two systems.
```

Comment:
```suggestion
*Description*: TLS is an initialism for Transport Layer Security, which succeeds the Secure Sockets Layer (SSL) protocol. TLS is a cryptographic protocol that uses the Public Key Infrastructure (PKI) method to encrypt network traffic between two systems. PKI uses symmetric encryption during a TLS handshake process to authenticate the connection between two systems.
```
Consider editing to avoid having three sentences in a row that begin with `TLS`.

Reply:
Hi @bburt-rh . This PR is marked as Work-in-Progress (WIP), so the content will change based on SME feedback. I hope to have updates soon.

Reply:
:heavy_check_mark: